core/templates/dev/head/pages/exploration_editor/editor_tab/teach_oppia_modal_directive.html

```diff
@@ -1,55 +1,66 @@
-<div class="modal-header">
-  <h3>Enter a Sample Answer</h3>
-</div>
+<style>
+.unresolved-answers-loading-text {
+  font-size: 1.17em;
+  text-align: center;
+}
 
-<div class="modal-body">
-  <div class="oppia-rule-details-header">
-    <test-interaction-panel state-content="stateContent" input-template="inputTemplate" on-submit-answer="submitAnswer(answer)">
-    </test-interaction-panel>
+.unresolved-answer {
+  border: black solid 1px;
+  border-radius: 5px;
+  padding: 1%;
+  margin-top: 1%;
+}
 
-    <div ng-if="trainingDataAnswer" style="padding-top: 10px; border-top: 1px solid #e5e5e5;">
-      <form name="testOppiaForm" class="form-inline">
-        <training-panel ng-if="classification.answerGroupIndex >= 0" answer="trainingDataAnswer" answer-feedback="trainingDataFeedback" answer-outcome-dest="trainingDataOutcomeDest" classification="classification" on-finish-training="finishTeaching(true)">
-        </training-panel>
-      </form>
-      <div ng-if="classification.answerGroupIndex === -1">
-        <div>
-          <span><strong>If Oppia encounters the answer:</strong></span>
-        </div>
+.unresolved-answer-feedback {
+  display: inline-block;
+  color: #00897B;
+}
 
-        <div>
-          <div angular-html-bind="answerTemplate">
-          </div>
-        </div>
+.unresolved-answer-response {
+  display: inline-block;
+  width: 75%;
+}
 
-        <br>
+.unresolved-answer-btn {
+  display: inline-block;
+  text-align: right;
+}
+</style>
 
-        <div>
-          <span><strong>it will reply with:</strong></span>
-        </div>
+<div class="modal-header">
+  <h3>Unresolved Answer</h3>
+</div>
 
-        <div>
-          <div angular-html-bind="trainingDataFeedback">
-          </div>
+<div class="modal-body">
+  <div class="unresolved-answers-list oppia-rule-details-header">
+    <p class='unresolved-answers-loading-text' ng-show="showLoadingDots">Loading <loading-dots></loading-dots></p>
+    <div ng-show="unresolvedAnswers.length > 0" style="padding-top: 10px; border-top: 1px solid #e5e5e5;">
+      <div class='unresolved-answer' ng-repeat="answer in unresolvedAnswers track by $index">
```

Comment:
Here and elsewhere, use double quotes for HTML attrs

/cc @apb7 @kevinlee12

Reply:
Filed Issue #5108 for the same. Thanks!

Reply:
Done.